test/scaleTest.js

```diff
@@ -0,0 +1,39 @@
+/* global suite test assert */
+
+function isAnimationEqual (keyframesA, keyframesB, currentTime) {
```

Comment:
Instead of taking a currentTime parameter this function could iterate over a bunch of currentTime values.